src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/error/CacheException.java

```diff
@@ -54,6 +54,21 @@ public CacheException(ErrorType errorType, String errorMessage, String errorDesc
         this.errorDescription = errorDescription;
     }
 
+    /**
+     * Instance constructor.
+     *
+     * @param errorType        error type
+     * @param errorMessage     error message
+     * @param errorDescription error description
+     * @param throwable        error catch
+     */
+    public CacheException(ErrorType errorType, String errorMessage, String errorDescription, Throwable throwable) {
```

Comment:
Why for we need it? Are you going to serialize traceback object and transfer it between kilda services?

Reply:
This was needed for testing. This change has been reverted.